srml/support/procedural/src/lib.rs

```diff
@@ -79,6 +79,17 @@ use proc_macro::TokenStream;
 ///   If the keys are not trusted (e.g. can be set by a user), a cryptographic `hasher` such as
 ///   `blake2_256` must be used. Otherwise, other values in storage can be compromised.
 ///
+///    Once read from the trie (e.g. through an RPC call), a tuple of `(T, Linkage<K>)` is returned,
+///    where `T` is the value and `K` is the key type of the mapping.
+///    [`Linkage`](../srml_support/storage/generator/struct.Linkage.html) is a pointer to the
+///    previous and the next element.
+///
+///    Each linked map has a special key in the tire that can be queried and it can be particularly
+///    useful to query all items: `$hash(head of module_name ++ " " ++ storage_name`). This will
```

Comment:
actually this is not hashed but maybe this is an issue ? like someone could target this if module_name nad storage_name are short ?
```suggestion
///    useful to query all items: `"head of  " ++ module_name ++ " " ++ storage_name`). This will
```

Reply:
According to the current code it is not hashed and I wrote it by mistake in the doc 👍